OpenKeychain/src/main/java/org/sufficientlysecure/keychain/remote/OpenPgpService.java

```diff
@@ -151,19 +149,17 @@ private KeyIdResult returnKeyIdsFromEmails(Intent data, String[] encryptionUserI
                         long id = cursor.getLong(cursor.getColumnIndex(KeyRings.MASTER_KEY_ID));
                         keyIds.add(id);
                     } else {
-                        missingUserIdsCheck = true;
                         missingEmails.add(email);
                         Log.d(Constants.TAG, "user id missing");
                     }
                     // another entry for this email -> two keys with the same email inside user id
                     if (cursor != null && cursor.moveToNext()) {
```

Comment:
btw: shouldn't this be a loop? If we have 3 keys with the same email address, all 3 should be pre-selected.